Sources/Resources/Localizable.strings

```diff
@@ -1,91 +0,0 @@
-/*
-  Localizable.strings
```

Comment:
Do we've multiple copies of `Resources` in Sources folder?

Reply:
No, just one with the assets in it.